### ISSUE TITLE
invalid jsonConfig in console

### ISSUE THREAD

**Issue by CherryKing93** [Homemade-Disaster/ioBroker.netatmo-energy#18]
**Describe the bug**  
The netatmo-energy Plugin isn't working. I don't have my smart thermostat in the Overview.
I have a warning in the console

**To Reproduce**  
Just start the Plugin

**Expected behavior**  
See my smart thermostat in the Admin-Tab.

**Screenshots & Logfiles**  
Console warning:
`netatmo-energy has an invalid jsonConfig: [{"instancePath":"/items/tabLogon/items/authenticate","schemaPath":"#/definitions/sendToProps/additionalProperties","keyword":"additionalProperties","params":{"additionalProperty":"window"},"message":"must NOT have additional properties"},{"instancePath":"/items/tabLogon","schemaPath":"#/properties/items/patternProperties/%5E.%2B/allOf/8/if","keyword":"if","params":{"failingKeyword":"then"},"message":"must match \"then\" schema"}]`

Screenshot:
![image](https://github.com/Homemade-Disaster/ioBroker.netatmo-energy/assets/37928149/bf916485-0f7f-4672-ac70-a45df675e712)

![image](https://github.com/Homemade-Disaster/ioBroker.netatmo-energy/assets/37928149/d1fe56f4-b811-4105-a26a-f68f01c16d9e)

**Versions:**  
 - Adapter version: 2.7.0
 - JS-Controller version: 5.0.17
 - Node version: v18.13.0
 - Operating system: ubuntu 23.10 in proxmox 8.1.3

**Comment by Homemade-Disaster** [comment]
Hi,

I do not have this problem with your reported software versions. Was it possible for you to authenticate using the button "AUTHENTIFIZIEREN SIE SICH BEI NETATMO"? Do you get data from the cloud (actual data in the datapoints for adapter netatmo-energy)?

**Comment by CherryKing93** [comment]
Hi,

The authentication worked and is OK:
![O Auth OK](https://github.com/Homemade-Disaster/ioBroker.netatmo-energy/assets/37928149/db643ebb-94c5-465d-9af8-4df4cdd0c4de)

Even in ioBroker there is an “OK”:
![OK](https://github.com/Homemade-Disaster/ioBroker.netatmo-energy/assets/37928149/8f7f0a83-fc37-4914-acdc-48bbafb64d0a)

Nothing can be seen in the objects. Everything is at "(null)"
![Objekte](https://github.com/Homemade-Disaster/ioBroker.netatmo-energy/assets/37928149/1e99d9fa-2118-4d14-aafe-f8ae7c2ed895)

**Comment by Homemade-Disaster** [comment]
Hi,

please try to install V2.7.2 and retest the adapter.

**Comment by CherryKing93** [comment]
Wow.
Thanks for the fast Update.
I will try it tomorrow and give an update afterwards.

**Comment by CherryKing93** [comment]
I testet it.
I don't have any thing in Objects.

This is in the Console:
```
2024-01-19 17:40:50.395	warn	netatmo-energy has an invalid jsonConfig: [{"instancePath":"/items/tabLogon/items/Information/size","schemaPath":"#/patternProperties/%5E.%2B/allOf/4/then/properties/size/type","keyword":"type","params":{"type":"number"},"message":"must be number"},{"instancePath":"/items/tabLogon","schemaPath":"#/properties/items/patternProperties/%5E.%2B/allOf/8/if","keyword":"if","params":{"failingKeyword":"then"},"message":"must match \"then\" schema"}]
```

**Comment by Homemade-Disaster** [comment]
Hi,

please set the debug level to silly and send me the log.

kr

**Comment by xpunkt-torie** [comment]
Hi!
I have the same issue on v2.7.0.
No erros. Only the "netatmo-energy has an invalid jsonConfig:" warning in log.
Auth is ok.

**Comment by Homemade-Disaster** [comment]
Please try to change your setup. Check the last two options ...
<img width="956" alt="image" src="https://github.com/Homemade-Disaster/ioBroker.netatmo-energy/assets/37286228/435bb0ce-9dc4-4f65-9d9a-1bde95a5ee78">

**Comment by xpunkt-torie** [comment]
Even with these settings I do not get any data displayed.

When looking at the logs, you can see that the API request 
`"https://api.netatmo.com/api/homesdata"`
 is then logged with many 
`"Event triggered: ...."`.
On the other hand, the call 
`"https://api.netatmo.com/api/homestatus" `
is also logged with 
`"Netatmo API-Status:200"` 
but then with many 
`"Search can't be optimized because wildcard not at the end, fallback to keys!: function(doc) { if (doc.type === 'state') emit(doc._id, doc) }"`.

If you call up the URL (incl. token, etc.) directly in a browser, data is also returned.

**Comment by CherryKing93** [comment]
This is my silly-log:

`netatmo-energy.0
2024-01-25 05:30:14.501	debug	API Request homestatus jeweils an API gesendet 180sec

netatmo-energy.0
2024-01-25 05:27:15.058	debug	Search can't be optimized because wildcard not at the end, fallback to keys!: function(doc) { if (doc.type === 'state') emit(doc._id, doc) }

netatmo-energy.0
2024-01-25 05:27:15.027	debug	API-Anfrage beendet

netatmo-energy.0
2024-01-25 05:27:15.027	debug	Search can't be optimized because wildcard not at the end, fallback to keys!: function(doc) { if (doc.type === 'state') emit(doc._id, doc) }

netatmo-energy.0
2024-01-25 05:27:15.027	debug	Search can't be optimized because wildcard not at the end, fallback to keys!: function(doc) { if (doc.type === 'state') emit(doc._id, doc) }

netatmo-energy.0
2024-01-25 05:27:15.027	debug	Search can't be optimized because wildcard not at the end, fallback to keys!: function(doc) { if (doc.type === 'state') emit(doc._id, doc) }

netatmo-energy.0
2024-01-25 05:27:15.026	debug	Search can't be optimized because wildcard not at the end, fallback to keys!: function(doc) { if (doc.type === 'state') emit(doc._id, doc) }

netatmo-energy.0
2024-01-25 05:27:15.026	debug	Search can't be optimized because wildcard not at the end, fallback to keys!: function(doc) { if (doc.type === 'state') emit(doc._id, doc) }

netatmo-energy.0
2024-01-25 05:27:15.026	debug	Search can't be optimized because wildcard not at the end, fallback to keys!: function(doc) { if (doc.type === 'state') emit(doc._id, doc) }

netatmo-energy.0
2024-01-25 05:27:15.024	debug	Netatmo API-Status:200

netatmo-energy.0
2024-01-25 05:27:14.797	debug	Anfrage: https://api.netatmo.com/api/homestatus?access_token=MYTOKEN&home_id=MYID

netatmo-energy.0
2024-01-25 05:27:14.796	info	API-Anfrage starten: homestatus`

**Comment by Homemade-Disaster** [comment]
Hi,

I'm back from vacation and was able to reproduce the error.
The issue is fixed in v2.7.3. 
Please install this version (already published) and test the issue again.

Everything should work now.

**Comment by CherryKing93** [comment]
Hope, the vacation was restful.
But it seams so. Now it works perfectly.
Thank you for the perfekt support.

![image](https://github.com/Homemade-Disaster/ioBroker.netatmo-energy/assets/37928149/f40eea6b-1817-4232-bd63-5de821de1e19)
